backends/tfhe-cuda-backend/cuda/include/integer.h

```diff
@@ -375,8 +375,8 @@ template <typename Torus> struct int_radix_lut {
     for (int i = 0; i < num_radix_blocks; i++)
       h_lwe_indexes[i] = i;
 
-    cuda_memcpy_to_gpu(lwe_indexes, h_lwe_indexes,
-                       num_radix_blocks * sizeof(Torus));
+    cuda_memcpy_async_to_gpu(lwe_indexes, h_lwe_indexes,
+                             num_radix_blocks * sizeof(Torus), stream);
     free(h_lwe_indexes);
```

Comment:
Is that ok ? you launch an async ops, then free.

What if the async ops is not finished before the `free` ?

Reply:
You're right I'm missing a sync stream here.